public/index.js

```diff
@@ -187,7 +187,7 @@ function signUp(newUser) {
 function userLogin(loginAttempt) {
     console.log("FROM USERLOGIN FUNCTION: ")
     console.log(loginAttempt)
-    $.ajax({ url: "/api/user/login", data: loginAttempt, method: "POST" })
+    $.ajax({ url: "/user/login", data: loginAttempt, method: "POST" })
```

Comment:
@AnthonyGaimari Please add /api back to the front of this ajax URL and then push back up and I can get it merged.